frontend/packages/gitops-plugin/src/components/list/GitOpsTableRow.tsx

```diff
@@ -3,32 +3,110 @@ import i18n from '@console/internal/i18n';
 import * as classNames from 'classnames';
 import { GitOpsAppGroupData } from '../utils/gitops-types';
 import { RowFunction, TableData, TableRow } from '@console/internal/components/factory';
-import { Button } from '@patternfly/react-core';
-import { history, ExternalLink } from '@console/internal/components/utils';
+import { Button, Flex, FlexItem, Split, SplitItem, Tooltip } from '@patternfly/react-core';
+import { history, ExternalLink, Timestamp } from '@console/internal/components/utils';
 import { routeDecoratorIcon } from '@console/dev-console/src/components/import/render-utils';
+import {
+  GreenCheckCircleIcon,
+  YellowExclamationTriangleIcon,
+  GrayUnknownIcon,
+} from '@console/shared';
+import './GitOpsTableRow.scss';
 
 const tableColumnClasses = [
   classNames('pf-m-width-20'), // Application name
-  classNames('pf-m-width-40'), // Git repository
+  classNames('pf-m-width-30'), // Git repository
   classNames('pf-m-hidden', 'pf-m-visible-on-md', 'pf-m-width-20'), // Environments
-  classNames('pf-m-hidden', 'pf-m-visible-on-lg', 'pf-m-width-20'), // Last deployment
+  classNames('pf-m-hidden', 'pf-m-visible-on-lg', 'pf-m-width-30'), // Last deployment
 ];
 
-const handleClick = (appGroup: GitOpsAppGroupData) => {
+const handleClick = (appGroup) => {
```

Comment:
Avoid removing types. Always swap them out for another one. If this is not the right type, we need to swap it out for another one or update the type. 

This may be moot for the `Link` implementation -- but if you keep a stand alone method for crafting the link url, it needs types. Stand-alone methods always need types as there is no TS smart typing possible.

Reply:
Ah, already requested by Regina.

Reply:
Done.